frontend/src/pages/simulationWindow.js

```diff
@@ -23,22 +23,26 @@ function SimulationWindow() {
   const [pages, setPages] = useState({
     introduction: { visited: true, completed: true, pageNumber: 0, html: (<Introduction />) },
     projectAssignment: { visited: false, completed: true, pageNumber: 1, html: (<ProjectAssignment />) },
-    initialReflection: { visited: false, completed: true, pageNumber: 2, html: (<Reflection
+    initialReflection: { visited: true, completed: true, pageNumber: 2, html: (<Reflection
       content_url="/scenarios/initialReflection" res_url="/scenarios/initialReflection/response" nextPageID="initialAction" prevPageID="projectAssignment" title="Reflect on Initial Information"/>) },
-    initialAction: { visited: false, completed: false, pageNumber: 3, html: (<InitialAction />) },
+    initialAction: { visited: true, completed: true, pageNumber: 3, html: (<Action
+      content_url="/scenarios/initialAction" nextPageID="gatheredInformation" prevPageID="initialReflection" title="Initial Action"/>) },
     gatheredInformation: { visited: false, completed: false, pageNumber: 4, html: (<GatheredInformation />) },
     stakeholders: { visited: false, completed: true, pageNumber: 5, html: (<Stakeholders />) },
-    middleReflection: { visited: false, completed: true, pageNumber: 6, html: (<Reflection
+    middleReflection: { visited: true, completed: true, pageNumber: 6, html: (<Reflection
```

Comment:
The reflection and action pages should have `visited: false`

Reply:
Fixed thanks!